scripts/submit_csv.sh

```diff
@@ -28,12 +28,32 @@ shift
 TARGET=${1:-2}
 shift
 
+# environment variable to indicate whether the job is running on condor or slurm
+SYSTEM=${SYSTEM:-condor}
+
 # process csv file into jobs
-CSV_FILE=$($(dirname $0)/csv_to_chunks.sh ${FILE} ${TARGET})
+if [ -n "${CSV_FILE:-}" ]; then
+  # allow to set custom csv file for job instead of fetching from web archive
+  CSV_FILE=$(realpath -e ${CSV_FILE})                     
+else
+  CSV_FILE=$($(dirname $0)/csv_to_chunks.sh ${FILE} ${TARGET})
+fi
 
 # create command line
-EXECUTABLE="./scripts/run.sh"
-ARGUMENTS="${TYPE} EVGEN/\$(file).\$(ext) \$(nevents) \$(ichunk)"
+EXECUTABLE="$PWD/scripts/run.sh"
```

Comment:
Probably should use something that determines the directory of the current script, e.g. `SCRIPTDIR=$(dirname $0)` or so. Then this script can be run from outside the directory, and the files it generates can be better organized.